java/tools/slice-tools/README.md

```diff
@@ -0,0 +1,193 @@
+# Ice Slice Tools Gradle Plugin
+
+The **Ice Slice Tools Gradle Plugin** integrates the **Slice-to-Java compiler (`slice2java`)** into Gradle-based
+**Java** and **Android** projects. It automates the compilation of Slice (`.ice`) files into Java source files and
+integrates seamlessly with the Gradle build system.
```

Comment:
Should we add a note like: it replaces the Ice Builder Gradle plugin (use official name). While you can still use the Builder with 3.8, we recommend using the Slice tools with Ice 3.8.